packages/react/src/components/TearSheet/TearSheet.story.jsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 5737-M66, 5900-AAA
+ * (C) Copyright IBM Corp. 2020 All Rights Reserved.
+ * US Government Users Restricted Rights - Use, duplication, or disclosure
+ * restricted by GSA ADP Schedule Contract with IBM Corp.
+ */
+
+import React, { useState } from 'react';
+import { withKnobs } from '@storybook/addon-knobs';
+
+import Button from '../Button';
+
+import TearSheetWrapper from './TearSheetWrapper';
+import TearSheet from './TearSheet';
+
+export const Default = () => <TearSheet1 />;
+export default {
+  title: 'TearSheet',
+  decorators: [withKnobs],
+  component: Default,
+};
+
+const TearSheet1 = () => {
+  const [isOpen, setOpen] = useState(false);
+
+  return (
+    <>
+      <Button style={{ margin: '8rem 0 0 8rem' }} onClick={() => setOpen(true)}>
+        Show TearSheet
+      </Button>
+
+      <TearSheetWrapper isOpen={isOpen} onCloseAllTearSheets={() => setOpen(false)}>
+        <TearSheet
+          title="First sheet"
+          description="Generic description"
+          onClose={() => setOpen(false)}
+        >
+          {(idx, openNextSheet, goToPreviousSheet) => (
+            <div
+              style={{
+                width: '1000px',
+                justifyContent: 'space-between',
+                display: 'flex',
+                flexDirection: 'column',
+                height: '100%',
+              }}
+            >
+              <Button
+                kind="tertiary"
+                onClick={openNextSheet}
+                style={{ margin: '2rem 0 0 2rem', maxWidth: '13rem' }}
+              >
+                Open 2nd sheet
+              </Button>
+              <div
+                style={{
+                  display: 'flex',
+                  justifyContent: 'flex-end',
+                  borderTop: '1px solid #e0e0e0',
+                }}
+              >
+                <Button
+                  onClick={() => goToPreviousSheet()}
```

Comment:
Can we also fire the onClose event here? I know it's just the story, but currently, if you click the "Close" button (not the close 'x') on the first tearsheet, clicking the "Show TearSheet" button will not show it anymore.

Reply:
The `onClose` event is only passed to the `TearSheet` component to be used by the `x` button, but I've resolved this issue by adding the `setOpen(false)` to the onClick event of this particular button.